_config/config.yml

```diff
@@ -6,9 +6,18 @@ SilverStripe\Core\Injector\Injector:
     class: SilverStripe\Control\SimpleResourceURLGenerator
     properties:
       NonceStyle: mtime
+  SilverStripe\Core\Manifest\PrioritySorter.modulesorter:
+    class: SilverStripe\Core\Manifest\PrioritySorter
+    properties:
+      RestKey: 'other_modules'
 SilverStripe\Control\HTTP:
   cache_control:
     max-age: 0
     must-revalidate: "true"
     no-transform: "true"
   vary: "Cookie, X-Forwarded-Protocol, User-Agent, Accept"
+SilverStripe\Core\Manifest\ModuleManifest:
+  project: ''
+  module_priority:
+    - 'other_modules'
```

Comment:
Discussed outside this thread, but we should figure out if top-to-bottom or bottom-to-top is more semantically correct.